packages/fast-components-react-base/src/hypertext/hypertext.props.ts

```diff
@@ -0,0 +1,23 @@
+import * as React from "react";
+
+/**
+ * Hypertext HTML tags
+ */
+export enum HypertextHTMLTags {
+    a = "a",
+}
+
+export interface IHypertextProps {
+
+    id?: string;
+
+    text?: string;
+
+    href?: string;
+
+    ariaLabel?: string;
+
+    onClick?: (event?: React.MouseEvent<HTMLElement>) => void;
```

Comment:
This will pass through if someone passes it. You can remove this as a first class prop